src/config/delorean.config.ts

```diff
@@ -12,6 +14,11 @@ export const SiteTheme = {
   SponsorHeader: {
     backgroundColor: '#6200EE',
     color: '#fff'
+  },
+  HeroBackground: {
+    backgroundImage: `url(${background})`,
+    backgroundSize: 'cover',
+    backgroundPosition: 'center',
```

Comment:
If these are meant to always be size 'cover' and position 'center' then maybe this should be inside of the `.intro` class instead of inside of site configuration?